runtime/compiler/runtime/JITServerAOTCache.cpp

```diff
@@ -812,13 +890,70 @@ JITServerAOTCacheMap::get(const std::string &name, uint64_t clientUID)
       return NULL;
       }
 
+   // See if we can load the cache from disk
+   TR::CompilationInfo *compInfo = TR::CompilationInfo::get();
+   if (compInfo->getPersistentInfo()->getJITServerUseAOTCachePersistence() &&     // Persistence feature is enabled
+       _cachesExcludedFromLoading.find(name) == _cachesExcludedFromLoading.end()) // I am allowed to load this cache from disk
+      {
+      if (TR::Options::getVerboseOption(TR_VerboseJITServer))
+            TR_VerboseLog::writeLineLocked(TR_Vlog_JITServer, "AOT cache: See if we can load cache %s from disk\n", name.c_str());
```

Comment:
This vlog message is redundant since we log the result of the operation on all code paths down from here.